AggregateMetrics/AggregateMetrics/AzureWebApp/FlexiblePerformanceCounterGauge.cs

```diff
@@ -35,5 +35,19 @@ public MetricTelemetry GetValueAndReset()
 
             return metric;
         }
+
+        /// <summary>
+        /// Returns the current value of the counter and resets the metric associated with the gauge. This method is for testing purposes only.
+        /// </summary>
+        /// <returns> Metric Telemetry object</returns>
+        public MetricTelemetry GetValueAndResetHttp()
```

Comment:
Don't do "special" case for Http. This method is an implementation of an interface and aggregation module will use it to get the value and send